AHK/src/hider/ahkhider.ahk

```diff
@@ -0,0 +1,78 @@
+﻿/*
```

Comment:
Will save the code, but will not automatically include it into the bin or exe file. People can choose whether to run it or not.